kin-devplatform-sdk/src/main/java/kin/devplatform/data/blockchain/BlockchainSourceImpl.java

```diff
@@ -261,17 +262,17 @@ private void incrementBalanceCount() {
 	}
 
 	private void startBalanceListener() {
-		Logger.log(new Log().withTag(TAG).text("startBalanceListener"));
-		balanceRegistration = account.blockchainEvents()
-			.addBalanceListener(new EventListener<kin.core.Balance>() {
-				@Override
-				public void onEvent(kin.core.Balance data) {
-					final double prevBalance = balance.getValue().getAmount().doubleValue();
-					setBalance(data);
-					eventLogger.send(KinBalanceUpdated.create(prevBalance));
-				}
-			});
-	}
+			Logger.log(new Log().withTag(TAG).text("startBalanceListener"));
+			balanceRegistration = account.blockchainEvents()
+				.addBalanceListener(new EventListener<kin.core.Balance>() {
+					@Override
+					public void onEvent(kin.core.Balance data) {
+						final double prevBalance = balance.getValue().getAmount().doubleValue();
+						setBalance(data);
+						eventLogger.send(KinBalanceUpdated.create(prevBalance));
+					}
+				});
+		}
```

Comment:
Format the code because i think you have an extra tab.
I saw it in other methods as well so maybe just format all the code in this class

Reply:
:+1: